src/app/shell/personal-cabinet/provider/create-workshop/create-description-form/create-description-form.component.html

```diff
@@ -60,7 +60,7 @@
       >{{ 'FORMS.PLACEHOLDERS.KEYWORDS_START' | translate }} {{ validationConstants.MAX_KEYWORDS_LENGTH }}
       {{ 'FORMS.PLACEHOLDERS.KEYWORDS_END' | translate }}</span
     >
-    <mat-chip-list #chipList>
+    <mat-chip-grid #chipGrid>
```

Comment:
It is better to use mat-chip-grid together with the input to follow the correct [interaction pattern](https://v16.material.angular.io/components/chips/overview#text-entry)